onecodex/viz/_bargraph.py

```diff
@@ -32,6 +32,7 @@ def plot_bargraph(
         height=None,
         group_by=None,
         link=Link.Ocx,
+        exclude_usermeta=False,
```

Comment:
@clausmith Ah, well the entire `usermeta` field would need to be excluded, and it _could_ be used for other things in the future (ie. specifying the name of a downloadable chart file). I figured it would be more precise to indicate exactly what it's doing, which is making sure the `usermeta` field doesn't exist on the chart, and more flexible in the future should we want to add any functionality that also makes use of it. But if it's unlikely we ever will, and if it's much clearer for users renamed to `link_target`, I'm fine with that. LMK what you think about this tradeoff.

Reply:
I think for now I'd err towards user-friendliness. If we end up `usermeta` a lot in the future, I think we could consider a more sophisticated solution (or patching `altair` to support our use case).

Reply:
I don't love the idea of an argument that is essentially a boolean but that we are passing a specific string into for one case and ignoring or erroring on other strings. What about a boolean `open_links_in_new_tabs` arg that is `True` by default and skips appending `usermeta` if `False`?